pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/RealtimeTableDataManager.java

```diff
@@ -354,7 +354,7 @@ private boolean isValid(Schema schema, IndexingConfig indexingConfig) {
       }
     }
     // 2. We want to get the schema errors, if any, even if isValid is false;
-    if (!SchemaUtils.validate(schema, _logger)) {
+    if (!SchemaUtils.validate(schema, false, _logger)) {
```

Comment:
Why skipping the fields validation?

Reply:
the schema used here is the one from zk metadata. The field names validation will already have been performed when the schema was added or updated. 
If the user skipped validating reserved keywords when adding/updating schema, then we will have reserved keywords in this schema. Hence, at this point, we should not enforce further field names validations, as we don't know if those are intentional or not.